packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -191,6 +192,9 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
 
   const cameraButtonIsEnabled = isEnabled(options?.cameraButton);
 
+  /* @conditional-compile-remove(raise-hand) */
+  const raiseHandButtonIsEnabled = true;
```

Comment:
Should this use the `isEnabled` function to ensure the options are applied?

Reply:
Yes, thanks